server_tests/test/clientTests.js

```diff
@@ -1528,50 +1539,28 @@ describe("RUST API TEST SUITE", function() {
                     })
                     .then( function onSuccess (res) {
                         Comparison.bublStopOutput(res);
-                    }, shouldNotGetHere);
+                        deferred.resolve();
+                    })
+                    .catch(deferred.reject);
```

Comment:
Well, it is true that anything calling into the OSC api requires the use of `wrapError`... In those cases it'd be more like `.catch(wrapError).then(deferred.resolve, deferred.reject)`